README.md

```diff
@@ -1,3 +1,5 @@
+[![Review Assignment Due Date](https://classroom.github.com/assets/deadline-readme-button-22041afd0340ce965d47ae6ef1cefeee28c7c493a6346c4f15d667ab976d596c.svg)](https://classroom.github.com/a/8wgCKhpZ)
+[![Open in Visual Studio Code](https://classroom.github.com/assets/open-in-vscode-2e0aaae1b6195c2367325f4f02e2d04e9abb55f0b24a779b69b11b9e10269abc.svg)](https://classroom.github.com/online_ide?assignment_repo_id=15706606&assignment_repo_type=AssignmentRepo)
 # se-day-2-git-and-github
 ## Explain the fundamental concepts of version control and why GitHub is a popular tool for managing versions of code. How does version control help in maintaining project integrity?
```

Comment:
Version Control is a system that allows us to track changes made to different versions of files over time. It also helps to ensure that multiple persons can collaborate on a project without overwriting each other's work.

Github is a popular tool for managing versions of code because it is built on top of git, and as such has amazing features such as, collaboration, centralized repository, Issue tracking, code reviews, community and open source, Integrations and so much more.

Version control helps in maintaining project integrity by 
- Tracking changes
- Project backup, thereby ensuring that even if local copy of code is lost or corrupted, the remote repository still has a history of the project.
- Collaboration, without overwriting each other's work on the same project.
- Accountability.